pylint/checkers/format.py

```diff
@@ -251,13 +252,13 @@ class FormatChecker(BaseTokenChecker, BaseRawFileChecker):
         ),
     )
 
-    def __init__(self, linter=None):
+    def __init__(self, linter: PyLinter):
         super().__init__(linter)
-        self._lines = None
-        self._visited_lines = None
-        self._bracket_stack = [None]
+        self._lines: dict[int, str] = {}
+        self._visited_lines: dict[str | None, int | None] = {}
+        self._bracket_stack: list[str | None] = [None]
```

Comment:
Does this do anything? It seems we only assign `[None]` to it, or am I missing something obvious?

Reply:
Good observation @DanielNoord. If you do `git show 480e0ad898` and view the module, there are some push operations to add values to `self._bracket_stack`, but as you say, we only assign `[None]` now. I'll remove it.